src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -140,6 +140,22 @@ public async Task SpecializeAsync()
             _logger.LogDebug("Completed language worker channel specialization");
         }
 
+        public async Task WorkerWarmup()
```

Comment:
We should have an `Async` suffix for all methods that use task, this is often forgotten but we should try and keep that practice going  `WorkerWarmupAsync`